vars/loadEnvFromString.groovy

```diff
@@ -0,0 +1,11 @@
+#!groovy
+
+def call(String parameter) {
+    if (parameter == null || parameter.isEmpty()) {
+        return
+    }
+    def props = readProperties text: parameter
+    props.each { key, value ->
+        env[key] = value
```

Comment:
Shouldn't we have some restricted env vars such as "cluster_backend", region name and some other for which we already have pipeline parameters?

Reply:
that won't be needed, those would be set again by the pipeline.

i.e. we set those first, and any pipeline setting something else, would win.

so it might be confusing to who gonna be trying to set those from this.

those are for things that doesn't have pipeline parameters.

to sum it up, I don't think there's a need for safegurads, time would tell,
at most, `use at you own cation !!!` on the top of it :)